common/model-views.cpp

```diff
@@ -3377,9 +3377,13 @@ namespace rs2
         syncer(viewer.syncer),
         _update_readonly_options_timer(std::chrono::seconds(6))
         , _detected_objects(std::make_shared< atomic_objects_in_frame >()),
-        _updates(viewer.updates),
-        _accuracy_health_model(*this, viewer)
+        _updates(viewer.updates)
     {
+        if (dev.is<device_calibration>())
+        {
+            _accuracy_health_model = std::unique_ptr<cah_model>(new cah_model(*this, viewer));
```

Comment:
Would be nice if there was std::make_unique()... check?

Reply:
There is, on C++14 on higher..